### PR TITLE
T2465/T2483 Fix DNS resolution from DHCP

### PR DESCRIPTION
**Pull request by kroy-the-rabbit**
This should migrate the pdns restart code from the DHCP update script to vyos-hostsd

### CODE REVIEW

**Review by c-po — APPROVED** [review]
Nice counting of work.

File: src/services/vyos-hostsd [hunk 1]
```diff
@@ -255,6 +263,13 @@ def handle_message(msg_json):
     with open(STATE_FILE, 'w') as f:
         json.dump(STATE, f)
 
+    if changes > 0:
+        (ret,return_code) = popen("pidof pdns_recursor");
```
Comment (c-po) [inline src/services/vyos-hostsd]: Why not use `vyos.util.process_named_running()` here?

Comment (kroy-the-rabbit) [reply src/services/vyos-hostsd]: Sounds good to me.  Also found a bug (which python probably would ignored)

**Comment by kroy-the-rabbit** [comment]
Royally messed up my commits here.  Resubmitting.

**Comment by c-po** [comment]
you can always git push —force

**Pull request closed without merging**